client/src/components/Applications/AppLayout/AppLayout.js

```diff
@@ -72,7 +84,12 @@ const AppsRoutes = () => {
           }}
         />
       ) : null}
-    </div>
+    </>
+      }
+      contentLayoutName="oneColumn"
+      contentShouldScroll
```

Comment:
To confirm, we want the entire apps view to scroll (new), not just the app form (current)?

Reply:
**Yes.** The chat was verbal, so I have no paper trail.

<details>
<summary>Logic</summary>

Once user selects an app, they don't need the app tray while reading info or filling out form (a user can only work on one app at a time), so scrolling it off-screen gives the user more space to read info and fill out form. In fact, Design wants to explore an app tray that is less obtrusive—maybe one that is only visible when necessary, not visible by default.

</details>